ext_emconf.php

```diff
@@ -9,7 +9,7 @@
     'description' => 'API to fetch translation units.',
     'category' => 'service',
     'shy' => 0,
-    'version' => '1.2.0',
+    'version' => '1.3.0',
```

Comment:
There is no functional change, only more support for TYPO3 versions, so I'd go fo 1.2.1 logically.